tests/integration/features/steps/integration_steps.py

```diff
@@ -479,11 +478,11 @@ def get_args(context, storage_provider, client_encryption, cassandra_url, use_mg
         )
 
     grpc_args = {
-        "enabled": grpc
+        "grpc_enabled": grpc
```

Comment:
No, this was actually a bug:
```python
>>> grpc_args = {'enabled': 1, 'grpc_option': 'foo'}
>>> k8s_args = {'enabled': 0, 'k8s_option': 'bar'}
>>> args = {**grpc_args, **k8s_args}
>>> args
{'enabled': 0, 'grpc_option': 'foo', 'k8s_option': 'bar'}
```
So the grcp enablement was getting the same key as the k8s one, and because k8s came second, it always won. As a result, the grpc server was not starting.
I suppose the args here are not directly matching the Medusa config, they are just used to build one somewhere else.